src/svc-notebooksd.yml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.0.0
+info:
+  title: notebooksd
+  version: 1.0.0
+servers:
+  - url: /
+paths:
+  /flows/orgs/{orgID}/flows:
+    $ref: './svc/notebooksd/paths/flows_orgs_orgID_flows.yml'
+  /flows/orgs/{orgID}/flows/{id}:
+    $ref: './svc/notebooksd/paths/flows_orgs_orgID_flows_id.yml'
```

Comment:
will these routes eventually be deprecated in favor of `/notebooks` and `/notebooks/{id}`?

Reply:
yes! They are temporary to make the transition from `flowd` to `notebooksd` easier. Once we move to `notebooksd`, we'll change the UI to use `/notebooks` and I'll remove these `/flows` routes.